apps/src/redux/sectionDataRedux.js

```diff
@@ -45,6 +46,7 @@ export const setSection = section => {
     id: section.id,
     script: section.script,
     students: sortedStudents,
+    codeReviewEnabled: section.code_review_enabled,
```

Comment:
nit: along the same lines as the above comment, I think it would make sense to just keep the naming here consistent, either way we decide - so  maybe `peerFeedbackEnabled` rather than `codeReviewEnabled`? my thinking here is that a `codeReviewEnabled` flag would more likely control hiding/showing the review tab entirely vs `peerFeedbackEnabled` seems to imply that we're just disabling the commenting on your peer's project functionality.

That being said, if it's a lot to rename, I don't think it's blocking for this PR :)